### DISCUSSION TITLE
Add Grid Support for Skin Layout Editor

### DISCUSSION THREAD

**Discussion by NotGumballer91** [ppy/osu#15044] [Ideas]
Since an actual grid is available for the editor, Wouldn't be nice to implement it in the Skin layout editor as well?
Manually dragging your components without any symmetry is quite annoying for certain users.

Here's a concept image
![grid](https://user-images.githubusercontent.com/64581009/136746873-e3272f46-3412-439c-98d8-1baf45d0b99e.png)

**Comment by peppy** [comment]
For sure, it definitely needs to be present in the skin editor. Hoping the implementation is pretty flexible and can just be dropped in.

**Reply by bdach** [reply]
Just displaying it should be a question of dropping it in. Snapping will be more involved to make work I imagine.
